doc/articles/get-started-rider.md

```diff
@@ -5,11 +5,11 @@ uid: Uno.GetStarted.Rider
 # Get Started on JetBrains Rider
```

Comment:
@jeromelaban I will let you double check these other documentation files mentioning Rider just in case:
- https://github.com/unoplatform/uno/blob/master/doc/articles/common-issues-rider.md
- https://github.com/unoplatform/uno/blob/master/doc/articles/get-started.md
- https://github.com/unoplatform/uno/blob/master/doc/.howto-template.md (wondering if this file is still needed/relevant)

Reply:
Thanks. The how-to template has not been used much, but it's still useful for the structure.